core/utils/thread_pool.hpp

```diff
@@ -21,7 +21,7 @@ namespace kagome {
    */
   class ThreadPool final : NonCopyable, NonMovable {
    public:
-    explicit ThreadPool(size_t thread_count)
+    explicit ThreadPool(size_t thread_count = 5ull)
```

Comment:
There can be multiple `ThreadPool`s, and their thread count will be configured in application_injector.cpp, because we know how many threads each component may need.

default=5 will allocate unnecessary threads and make debugging harder.